lib/supabase_auth_ui.dart

```diff
@@ -6,5 +6,6 @@ export 'src/components/supa_reset_password.dart';
 export 'src/components/supa_socials_auth.dart';
 export 'src/components/supa_phone_auth.dart';
 export 'src/components/supa_verify_phone.dart';
+export 'src/components/supa_native_auth.dart';
```

Comment:
Does this file exist?

Reply:
Just deleted it. Must have forgottent o remove it from the exports